pkg/kuttlctl/cmd/root.go

```diff
@@ -30,6 +32,7 @@ and serves as an API aggregation layer.
 		Version: version.Get().GitVersion,
 	}
 
+	cmd.PersistentFlags().StringVar(&k8s.ImpersonateAs, "as", "", "the username that you wish to impersonate")
```

Comment:
note as this is a `PersistentFlags` all dowstream commands will have access to this flag.